src/util.h

```diff
@@ -67,6 +67,14 @@ extern "C" {
 
 #define COMPILE_ERROR_ON(cond) ((void)sizeof(char[(cond) ? -1 : 1]))
 
+#define pf_always_inline __attribute__((always_inline)) inline
```

Comment:
Thats on of my problems with it, from the other macros, one would expect pf_always_inline defines \_\_attribute\_\_((always_inline))

But it is an exception, it also the the inline keyword.
It is obfuscation.